src/ecosystem/corals/Corals.jl

```diff
@@ -296,7 +296,10 @@ function create_coral_struct(bounds::Tuple{Float64,Float64}=(0.9, 1.1))::Nothing
         bounds=(0.25, 0.5),
         default_bounds=(0.25, 0.5),
         dists="unif",
-        name="Heritability", description="Heritability of DHW tolerance.")
+        criteria_keywords=(""),
+        name="Heritability",
+        description="Heritability of DHW tolerance.",
+    )
```

Comment:
A bit late now but you could have created your own separate constructor that defines a default value for these additional fields (`criteria_keywords`, `default_bounds`, etc) so you don't have to go around adding these fields to all use of `Param` types.

Leave it as is - we'll clean it up later.